src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work.");
```

Comment:
The message in `doWork` method is not informative enough. It should indicate that the 'Truck' is the type of `Machine` that is working now. For example: "Truck is doing its work."